mcs_benchmark_data/cli/commands/create_submission_pipeline_command.py

```diff
@@ -136,27 +114,9 @@ def make_submission_directories(
                 """The submission metadata files have been created at the following paths:\n 
                  - ./data/%s/submissions/submissions_metadata.jsonl\n 
                  - ./test_data/%s/submissions/submissions_metadata.jsonl\n
-            Edit the files by entering in the missing information that corresponds to the submission.\n""",
+            Edit the files by entering in the missing information that corresponds to the submission.""",

```

Comment:
The indentation will show up in the multiline string. You can do:
```
if some_program_construct_requiring_indent:
     f("""\
Unindented text line 1.
Unindented text line 2.""")
```

Above:
* The initial `"""` is at the appropriate indentation level for where you are in the code (or inside a `(` or whatever)
* The `\` right after `"""` says "don't include this first newline in the string".
* Subsequent newlines in the triple-quoted string end up in the string, so you don't need explicit `\n`.
* The body of the triple-quoted string itself is not indented, so indentation doesn't show up in the resulting literal.
* The triple quotes end after the last bit of actual text, so you don't have an additional newline at the bottom.